larpandoracontent/LArObjects/LArSupportVectorMachine.h

```diff
@@ -306,6 +319,24 @@ inline double SupportVectorMachine::CalculateClassificationScore(const DoubleVec
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+inline double SupportVectorMachine::CalculateProbability(const DoubleVector &features) const
+{
+    if (!m_enableProbability)
+    {
+        std::cout << "LArSupportVectorMachine: cannot calculate probabilities for this SVM" << std::endl;
+        return pandora::STATUS_CODE_NOT_INITIALIZED;
```

Comment:
This should raise an exception, rather than (presumably) cast the enumerated StatusCode into a double!